sign/ed25519/ed25519.go

```diff
@@ -163,6 +264,98 @@ func Verify(public PublicKey, message, signature []byte) bool {
 	return bytes.Equal(R, encR)
 }
 
+// Verify returns true if the signature is valid. Failure cases are invalid
+// signature, or when the public key cannot be decoded.
+// This function does not handle prehashed messages.
+func Verify(public PublicKey, message, signature []byte) bool {
```

Comment:
Let's match the name of these functions as follows:  
`Sign`  -  `Verify`
`SignPure`  -  `VerifyPure`
`SignPh`  -  `VerifyPh`
`SignWithCtx`  -  `VerifyWithCtx`